src/main/java/de/tum/in/www1/artemis/web/rest/ModelingSubmissionResource.java

```diff
@@ -105,24 +105,29 @@ public ModelingSubmissionResource(ModelingSubmissionService modelingSubmissionSe
      * submitting modeling submissions. The submit specific handling occurs in the ModelingSubmissionService.save() function.
      *
      * @param exerciseId         the id of the exercise for which to init a participation
+     * @param submissionId       the id of the submission which needs to be updated
      * @param principal          the current user principal
      * @param modelingSubmission the modelingSubmission to update
      * @return the ResponseEntity with status 200 (OK) and with body the updated modelingSubmission, or with status 400 (Bad Request) if the modelingSubmission is not valid, or
      *         with status 500 (Internal Server Error) if the modelingSubmission couldn't be updated
      */
-    @PutMapping("/exercises/{exerciseId}/modeling-submissions")
+    @PutMapping("/exercises/{exerciseId}/modeling-submissions/{submissionId}")
```

Comment:
This makes our whole REST API inconsistent. I think we should revert this change!

Reply:
Reverted